scripts/Dockerfile.verify

```
@@ -45,6 +45,11 @@ RUN curl -fsSL https://claude.ai/install.sh | bash
 SHELL ["/bin/sh", "-c"]
 ENV PATH="/home/testuser/.local/bin:${PATH}"
 
+# Install Codex CLI for the interactive Codex E2E harness.
+USER root
+RUN npm install -g @openai/codex
```

Hadolint (2.14.0)

[warning] Pin versions in npm. Instead of `npm install <package>` use `npm install <package>@<version>`

(DL3016)